commit 9f2d49a27efdfe941ed35753ceccf1c953667053
Author: Alan F <...>
Date:   Thu Apr 24 20:18:46 2014 +0100

    patch from rene.herman which fixes a type mismatch in some printf specifications in the transmission style cell renderer, causing 32bit to show 0 connected peers and unknown eta. thanks!

--- src/torrent-cell-renderer.c.orig	2014-04-24 20:32:59.851226237 +0200
+++ src/torrent-cell-renderer.c	2014-04-24 20:34:30.649059527 +0200
@@ -329,16 +329,16 @@
         if (priv->fileCount > 0) {
             g_string_append_printf(gstr,
                                    ngettext
-                                   ("Downloading from %1$li of %2$li connected peer",
-                                    "Downloading from %1$li of %2$li connected peers",
+                                   ("Downloading from %1$lli of %2$lli connected peer",
+                                    "Downloading from %1$lli of %2$lli connected peers",
                                     priv->webSeedsToUs + priv->peersToUs),
                                    priv->webSeedsToUs + priv->peersToUs,
                                    priv->webSeedsToUs + priv->connected);
         } else {
             g_string_append_printf(gstr,
                                    ngettext
-                                   ("Downloading metadata from %1$li peer (%2$s done)",
-                                    "Downloading metadata from %1$li peers (%2$s done)",
+                                   ("Downloading metadata from %1$lli peer (%2$s done)",
+                                    "Downloading metadata from %1$lli peers (%2$s done)",
                                     priv->connected + priv->webSeedsToUs),
                                    priv->connected + priv->webSeedsToUs,
                                    tr_strlpercent(buf,
@@ -348,8 +348,8 @@
     } else if (priv->flags & TORRENT_FLAG_SEEDING) {
         g_string_append_printf(gstr,
                                ngettext
-                               ("Seeding to %1$li of %2$li connected peer",
-                                "Seeding to %1$li of %2$li connected peers",
+                               ("Seeding to %1$lli of %2$lli connected peer",
+                                "Seeding to %1$lli of %2$lli connected peers",
                                 priv->connected), priv->peersFromUs,
                                priv->connected);
     }
